# sctp is NOT supported until kernel 4.10 (or later).  Currently RHEL 7.x has a 3.x kernel

--- packages/clip-selinux-policy/clip-selinux-policy/policy/modules/kernel/corenetwork.te.in	2018-04-10 14:09:40.900565046 -0400
+++ packages/clip-selinux-policy/clip-selinux-policy/policy/modules/kernel/corenetwork.te.in	2018-04-11 10:45:07.947490787 -0400
@@ -307,12 +307,12 @@
 portcon udp 1024-65535 gen_context(system_u:object_r:unreserved_port_t, s0)
 portcon tcp 1024-65535 gen_context(system_u:object_r:unreserved_port_t, s0)
 portcon tcp 512-1023 gen_context(system_u:object_r:hi_reserved_port_t, s0)
-portcon sctp 1024-65535 gen_context(system_u:object_r:unreserved_port_t, s0)
-portcon sctp 512-1023 gen_context(system_u:object_r:hi_reserved_port_t, s0)
+#portcon sctp 1024-65535 gen_context(system_u:object_r:unreserved_port_t, s0)
+#portcon sctp 512-1023 gen_context(system_u:object_r:hi_reserved_port_t, s0)
 portcon udp 512-1023 gen_context(system_u:object_r:hi_reserved_port_t, s0)
 portcon tcp 1-511 gen_context(system_u:object_r:reserved_port_t, s0)
 portcon udp 1-511 gen_context(system_u:object_r:reserved_port_t, s0)
-portcon sctp 1-511 gen_context(system_u:object_r:reserved_port_t, s0)
+#portcon sctp 1-511 gen_context(system_u:object_r:reserved_port_t, s0)
 
 ########################################
 #
